README.md

```diff
@@ -198,8 +197,8 @@ name. Since FOSS Heartbeat's model is trained to recognize empathy and praise
 as positive, and personal attacks as negative, we often have to shift the
 sentiment of specific words.
 
-You can use substitutions.txt to change word sentiment labels in the sentences
-from the default sentiment model. It involves stripping the '%' off the vim
+You can use [substitutions.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/language/substitutions.txt) to change word sentiment labels in the sentences
+from the default sentiment model. It involves stripping the '%' off the Vim
 substitution commands in substitution.txt, using the resulting file as a sed
```

Comment:
Done. I also standardized around this a bit through the rest of the doc, and found another typo in a file reference.